src/main/java/mate/academy/spring/model/dto/response/UserResponseDto.java

```diff
@@ -0,0 +1,40 @@
+package mate.academy.spring.model.dto.response;
+
+public class UserResponseDto {
+    private Long id;
+    private String email;
+    private String password;
+    private byte[] salt;
+
+    public Long getId() {
+        return id;
+    }
+
+    public void setId(Long id) {
+        this.id = id;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    public void setEmail(String email) {
+        this.email = email;
+    }
+
+    public String getPassword() {
+        return password;
+    }
+
+    public void setPassword(String password) {
```

Comment:
Remove the 'setPassword' method, as it's not a good practice to expose user passwords in response DTOs.